common/mariadb/values.yaml

```diff
@@ -30,8 +30,8 @@ db_performance_schema_instrument: # Enabling performance schema instruments.
   enabled: false
 
 root_password: null
-initdb_configmap: null
```

Comment:
pls note that this will require changes to all chart values too:
```
$ grep -nri initdb_configmap .
./cinder/values.yaml:130:  initdb_configmap: cinder-initdb
./manila/values.yaml:611:  initdb_configmap: manila-initdb
./octavia/values.yaml:56:  initdb_configmap: octavia-initdb
./barbican/values.yaml:61:  initdb_configmap: barbican-initdb
./designate/values.yaml:95:  initdb_configmap: designate-initdb
./nova/values.yaml:703:  initdb_configmap: true
./nova/values.yaml:743:  initdb_configmap: true
./nova/values.yaml:776:  initdb_configmap: true
./neutron/values.yaml:373:  initdb_configmap: neutron-initdb
./keystone/values.yaml:281:  initdb_configmap: keystone-initdb
./ironic/values.yaml:368:  initdb_configmap: ironic-mariadb-init
./glance/values.yaml:135:  initdb_configmap: glance-initdb
```

Reply:
we have as well in secret repository for neutron custom_init_configmap

Reply:
ticket created
https://jira.tools.sap/secure/RapidBoard.jspa?rapidView=37291&view=detail&selectedIssue=CCC-34498#